src/app/accounts/npub-by-username.ts

```diff
@@ -0,0 +1,17 @@
+import { checkedToUsername } from "@domain/accounts"
+import { CouldNotFindAccountError, RepositoryError } from "@domain/errors"
+import { AccountsRepository } from "@services/mongoose"
+import { Account } from "@services/mongoose/schema"
+
+export const npubByUsername = async (
+  username: Username,
+): Promise<{ npub: string; username: string } | ApplicationError> => {
```

Comment:
username can be assigned to type `Username` found [here](https://github.com/lnflash/flash/blob/a1362fd9525d4c1721a3fc937918cf376947a6eb/src/domain/primitives/index.types.d.ts#L3)